src/controllers/expensesController.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable no-console */
+const expenseService = require('../services/expensesServices.js');
+const { getById: getUserById } = require('../services/usersServices.js');
+
+const get = async (req, res) => {
+  const expenses = await expenseService
+    .getAll(req.query)
+    .then((exps) => exps.map(expenseService.normalize));
+
+  res.send(expenses);
+};
+
+const getOne = (req, res) => {
+  const expense = expenseService.normalize(req.entry);
+
+  res.send(expense);
+};
+
+const create = async (req, res) => {
+  const {
+    userId,
+    spentAt = new Date(),
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const user = await getUserById(userId);
+
+  if (!user) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const expense = await expenseService
+    .create(userId, spentAt, title, amount, category, note)
+    .then(expenseService.normalize);
```

Comment:
Inconsistent promise handling: Consider using `async/await` instead of mixing it with `.then()`. This will make the code more readable and maintainable.